airflow/kubernetes_executor_templates/basic_template.yaml

```diff
@@ -69,8 +69,8 @@ spec:
         defaultMode: 420
   restartPolicy: Never
   terminationGracePeriodSeconds: 30
-  serviceAccountName: airflow-worker-serviceaccount
-  serviceAccount: airflow-worker-serviceaccount
+  serviceAccountName: airflow-worker
+  serviceAccount: airflow-worker
```

Comment:
This change was needed after https://github.com/apache/airflow/commit/23768f694697be1b29ec00b8deeed5777d2538e8

However since we were not using the `pod_template_file` in the test we didn't find that it was broken